quick-start/e2e/specs/create/create.ts

```diff
@@ -190,7 +213,7 @@ export default function() {
       lastProperty = entityPage.lastProperty;
       entityPage.getPropertyName(lastProperty).sendKeys('price');
       entityPage.getPropertyType(lastProperty).element(by.cssContainingText('option', 'decimal')).click();
-      entityPage.getPropertyDescription(lastProperty).sendKeys('price description');
```

Comment:
why do we need to change it to "pricedesc"? The verification later is still "price description"